### PR TITLE
Feature/cart display component

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Review comments during the demo session has been incorporated. Please verify and let us know in case of any concern

### CODE REVIEW

File: core/src/main/java/com/hcl/ecomm/core/services/impl/CartServiceImpl.java [hunk 1]
```diff
@@ -79,4 +84,54 @@ public int getCartItemCount(String cartId) {
        cartItemCount = cartItemsArray.size();
        return cartItemCount;
     }
+
+    @Override
+    public String updateCartDetails(String payload) {
```
Comment (rishabh-behl) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/CartServiceImpl.java:89]: @DeepaliAgrwl : why is updateCartDetails method required ? you can reuse the same servlet Nithesh has created for cart item count

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/image/_cq_editConfig.xml [hunk 2]
```diff
@@ -23,8 +23,8 @@
             <plugins jcr:primaryType="nt:unstructured">
                 <crop
                     jcr:primaryType="nt:unstructured"
-                    supportedMimeTypes="[image/jpeg,image/png,image/webp,image/tiff]"
-                    features="*">
+                    features="*"
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/image/_cq_editConfig.xml]: @DeepaliAgrwl : what are the changes in Image component as part of this PR ?

File: core/src/main/java/com/hcl/ecomm/core/services/impl/CartServiceImpl.java (hunk 1 above)
Comment (DeepaliAgrwl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/CartServiceImpl.java:89]: @rishabh-behl  Nithesh is doing getcart and delete cart only. Updation is managed on locally on javascript. That is why I required this method

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/clientlibs/js/cartproductdisplay.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    }
+}
+
+
+$(document).ready(function (){
+   	 $('.order-price').text(totalBagPrice - bagDiscount);
+     $('.total-price').text(totalBagPrice - bagDiscount + deliveryCharges);
+	 placeOrderRedirection = $('.place-order-button').children().children().attr('href');
+	 $('.place-order-button').children().children().removeAttr("href");
+    //window.location.replace(placeOrderRedirection);
```
Comment (Rajamouli-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/clientlibs/js/cartproductdisplay.js]: why u need this comment line. please delete this.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/image/_cq_editConfig.xml (hunk 2 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/image/_cq_editConfig.xml]: @rishabh-behl  I have reverted back the changes for this xml

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/clientlibs/js/cartproductdisplay.js (hunk 3 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/cartproductdisplay/clientlibs/js/cartproductdisplay.js]: removed the commented line Rishabh

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**